cogs/qrchecker/qrchecker.py

```diff
@@ -15,10 +16,12 @@
 from redbot.core.bot import Red
 from redbot.core.commands import Context
 from redbot.core.config import Group
-from redbot.core.utils.chat_formatting import box, pagify
+from redbot.core.utils.chat_formatting import box, pagify, success
 
 KEY_ENABLED: str = "enabled"
+KEY_MAX_IMAGE_DIMENSIONS: str = "maxImageDimensions"
 
+BASE_GLOBAL: Dict = {KEY_MAX_IMAGE_DIMENSIONS: 80000000}
```

Comment:
How about `KEY_MAX_IMAGE_PIXELS`? Also I think type hinting for constants are not necessary but I'm refactoring this in a separate change as well so will have them removed there.

```suggestion
KEY_MAX_IMAGE_PIXELS: str = "maxImagePixels"

BASE_GLOBAL: Dict = {KEY_MAX_IMAGE_PIXELS: 80000000}
```